readme.md

```diff
@@ -1022,11 +1022,14 @@ For example, it can be helpful during development to avoid an infinite number of
 ###### pagination.stackAllItems
 
 Type: `boolean`\
-Default: `true`
+Default: `false`
+
+Defines how the property `allItems` in [`pagination.paginate`](#pagination.paginate), [`pagination.filter`](#pagination.filter) and [`pagination.shouldContinue`](#pagination.shouldContinue) is managed.
+
+By default, the property `allItems` is always an empty array.
 
-Defines how the parameter `allItems` in [pagination.paginate](#pagination.paginate), [pagination.filter](#pagination.filter) and [pagination.shouldContinue](#pagination.shouldContinue) is managed. When set to `false`, the parameter `allItems` is always an empty array.
+When set to `false`, the parameter `allItems` is always an empty array. This setting can impact memory usage when working with a large dataset.
```

Comment:
```suggestion
When set to `false`, the `allItems` parameter is always an empty array. If `true`, it can hugely increase memory usage when working with a large dataset.
```